crates/bevy_render/src/camera/bundle.rs

```diff
@@ -75,6 +75,23 @@ pub struct OrthographicCameraBundle {
 }
 
 impl OrthographicCameraBundle {
+    /// Create an orthographic projection camera to render 2D content.
+    ///
+    /// The projection creates a camera space where X points to the right of the screen,
+    /// Y points to the top of the screen, and Z points out of the screen (backward),
+    /// forming a right-handed coordinate system. The center of the screen is at `X=0` and
+    /// `Y=0`.
+    ///
+    /// The default scaling mode is [`ScalingMode::WindowSize`], resulting in a resolution
+    /// where 1 unit in X and Y in camera space corresponds to 1 logical pixel on the screen.
+    /// That is, for a screen of 1920 pixels in width, the X coordinates visible on screen go
+    /// from `X=-960` to `X=+960` in world space, left to right. This can be changed by changing
```

Comment:
Hmm, I haven't checked but the range [-960, 960] inclusively would be 1921 pixels. I wonder if for a dimension divisible by 2, the pixel coordinates are that their centres and so are in the range [-959.5, 959.5] inclusively and the centres in the x dimension are at every .5 value. Do you or does anyone know?

Reply:
I think possibly the pixel center is not aligned with the coordinates, but the pixel corners are. In any case, I didn't feel like digging into those details; this can be fixed once merged if someone knows the exact formula. I'd rather merge _some_ form of documentation soon since there's absolutely nothing, and some other parts contain important information that need to be documented.

Reply:
In any case, the phrasing "coordinates visible on screen" is already vague in what it refers to, so I don't feel there's a need to be more precise here. I just wanted to carry the general idea. Or, I'm happy to replace it if someone wants to suggest a better phrasing for the entire sentence.

Reply:
I guess we can roll with this and then correct it later.